src/main/java/ladder/domain/Line.java

```diff
@@ -11,20 +11,32 @@ public Line(List<Boolean> line) {
         this.line = line;
     }
 
-    public Line addPoint(boolean point) {
+    public Line addedLine(boolean point) {
         ArrayList<Boolean> newLine = new ArrayList<>(line);
         newLine.add(point);
         return new Line(newLine);
     }
 
-    public boolean isLastPointTrue() {
+    public boolean isLastTrue() {
         if (line.size() == 0) {
```

Comment:
```suggestion
        if (line.isEmpty()) {
```

이런식으로 제공해주는 것을 사용하면, 더 깔끔해 보일수 있겠어요!